### ISSUE TITLE
Why is `<text>` `@type` a SHOULD? Should it become a MUST or should this information be reserved for the `<schemaRef>` element

### ISSUE THREAD

**Issue by jeffreycwitt** [lombardpress/lombardpress-schema#120]
The Docs currently state the  `<text>` element SHOULD take an `@type` attribute the value of which is "critical", "diplomatic", or "translation".

This piece of information only seems needed for processing, but it becomes useless if only some of the documents record this information and others do not

It seems to me that either this information should be required or not present. Otherwise, it seems kind of useless. 

This would be a breaking change, so the MUST would have to be changed at the promotion to 2.0. 

But in the subsequent 1.x release, we could highly encourage its use and note that it will become required in the 2.0 release. 

There also seems to be some overlap between this information and the information provided in the schemaRef. 

Perhaps this information is better situated in the `<schemaRef>`.

**Comment by stenskjaer** [comment]
I agree that maybe `<schemaRef>` may be better, but we can't transition to only use that until we have a translation schema, I think. 

As it is now, since it is only **SHOULD**, I guess processors should prefer to determine the type by the schema referenced in `<schemaRef>`. So in a sense it may be just as good just to loose the `text@type` here and move entirely to reliance on the `<schemaRef>`. That makes it less ambiguous to processor developers.
